modules/theme-tools/social-menu.php

```diff
@@ -40,26 +40,61 @@ function jetpack_social_menu_init() {
 
 	// Enqueue CSS
 	add_action( 'wp_enqueue_scripts', 'jetpack_social_menu_style' );
+
+	// Load SVG icons related functions and filters
+	if ( 'svg' === jetpack_social_menu_get_type() ) {
+		require( dirname( __FILE__ ) . '/social-menu/icon-functions.php' );
+	}
 }
 add_action( 'after_setup_theme', 'jetpack_social_menu_init', 99 );
 
-/* Function to enqueue CSS */
+/**
+ * Return the type of menu the theme is using.
+ *
+ * @uses get_theme_support()
+ * @return $menu_type
```

Comment:
Please add return `type`.  in this case I think it would be 

`@return null|string $menu_type`